lib/mapObjects/MiscObjects.cpp

```diff
@@ -2200,12 +2196,6 @@ void CGLighthouse::initObj(CRandomGenerator & rand)
 	}
 }
 
-std::string CGLighthouse::getHoverText(PlayerColor player) const
```

Comment:
is current lighthouse string OK?

Reply:
yep, made screenshots on the test map after applying these changes

Reply:
oh, I am blind